src/control/lib/control/system.go

```diff
@@ -424,13 +457,17 @@ func StopRanks(ctx context.Context, rpcClient UnaryInvoker, req *RanksReq) (*Ran
 	})
 	rpcClient.Debugf("DAOS system stop-ranks request: %+v", req)
 
-	return rpcToRanks(ctx, rpcClient, req)
+	return invokeRPCFanout(ctx, rpcClient, req)
 }
 
 // ResetFormatRanks concurrently resets format state on ranks across all hosts
-// supplied in the request's hostlist. The function blocks until all
-// results (successful or otherwise) are received, and returns a
-// single response structure containing results for all host operations.
+// supplied in the request's hostlist.
+//
+// This is called from SystemResetFormat in server/ctl_system.go with a
+// populated host list in the request parameter and blocks until all results
```

Comment:
The RPC calls into the ctl_system.go function, right? Server isn't calling into the client API, is it?

Reply:
RPC triggered through client API from DMG hitting server endpoint handled in `server/ctl_system.go`. Then SystemResetFormat on the server will call the client API to invoke fanout to all the ranks (`ctl_system.go:SystemResetFormat()` initiates the host list for the invoker, `lib/control/system.go:ResetFormatRanks()` invokes RPCs across all the hosts)

Reply:
Ahhh, got it. Thanks.